scripts/visualise_results.py

```diff
@@ -49,16 +70,33 @@ def get_data_frame(results):
         for result in results
     ]
 
-    percentile_columns = (f"{percentile}th" for percentile in PERCENTILES_TO_GRAPH)
-    return DataFrame(result_fields, columns=["DATE", *percentile_columns])
+    return create_dataframe(result_fields, PERCENTILES_TO_PLOT)
+
+
+def get_additional_metrics_data_frame(results):
+    result_fields = [
+        [
+            result.date,
+            result.statistics.average_pdf_percentile,
+            result.statistics.average_session_percentile,
+        ]
+        for result in results
+    ]
+
+    return create_dataframe(result_fields, ADDITIONAL_METRICS_TO_GRAPH)
 
 
 if __name__ == "__main__":
     parsed_variables = parse_environment_variables()
     number_of_days = parsed_variables["number_of_days"]
 
     folders = sorted(glob(f"{parsed_variables['output_dir']}/*"))
-    results = get_results(folders, number_of_days)
-    dataframe = get_data_frame(results)
 
-    plot_data(dataframe, number_of_days)
+    results = list(get_results(folders, number_of_days))
+
+    performance_dataframe = get_performance_data_frame(results)
+    additional_metrics_dataframe = get_additional_metrics_data_frame(results)
+
+    dataframe_values = [performance_dataframe, additional_metrics_dataframe]
```

Comment:
Should this just be `dataframes`? Do we need this var since it's only used once, maybe would be clearer if the method call used the argument name?

Reply:
I kept the var in so it makes the method call a bit clearer, happy to just put the array in the method call, I updated the vars to keep the names consistent 👍